03 Tests samples/src/components/NameForm/NameForm.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { NameForm } from './NameForm';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
We can declare an configure adapter in config files instead of declare inside each test file separated

Reply:
DOne